source/common/runtime/runtime_features.cc

```diff
@@ -41,18 +41,19 @@ constexpr const char* runtime_features[] = {
 constexpr const char* disallowed_features[] = {
     // Acts as both a test entry for deprecated.proto and a marker for the Envoy
     // deprecation scripts.
-    "envoy.deprecated_features.deprecated.proto:is_deprecated_fatal",
     "envoy.deprecated_features.bootstrap.proto:runtime",
-    "envoy.deprecated_features.redis_proxy.proto:catch_all_cluster",
-    "envoy.deprecated_features.lds.proto:use_original_dst",
-    "envoy.deprecated_features.server_info.proto:max_stats",
-    "envoy.deprecated_features.redis_proxy.proto:cluster",
-    "envoy.deprecated_features.server_info.proto:max_obj_name_len",
     "envoy.deprecated_features.config_source.proto:UNSUPPORTED_REST_LEGACY",
+    "envoy.deprecated_features.deprecated.proto:is_deprecated_fatal",
     "envoy.deprecated_features.ext_authz.proto:use_alpha",
-    "envoy.deprecated_features.route.proto:enabled",
     "envoy.deprecated_features.fault.proto:type",
+    "envoy.deprecated_features.lds.proto:use_original_dst",
+    "envoy.deprecated_features.redis_proxy.proto:catch_all_cluster",
+    "envoy.deprecated_features.redis_proxy.proto:cluster",
+    "envoy.deprecated_features.route.proto:enabled",
     "envoy.deprecated_features.route.proto:runtime_key",
+    "envoy.deprecated_features.server_info.proto:max_obj_name_len",
+    "envoy.deprecated_features.server_info.proto:max_stats",
+    "envoy.deprecated_features.v1_filter_json_config",
```

Comment:
Do we want to alphabetize or organize per release?

Reply:
I'm up for either, happy to put in comments per-release, or just alpha the whole thing. LMK what you prefer.

Reply:
I think I'd prefer comments per-release, or changing the scrip to do alpha order so we don't have to do it manually each time.  Happy to go either way.

Reply:
OK I will fix.